.npmignore

```diff
@@ -1,20 +1,8 @@
-# Configuration files
+# Development-specific files
 .deploy.env
 .editorconfig
 .eslintrc.js
 .gitattributes
 .gitignore
 .travis.yml
-
-# Tests
-test-utils/
-tests/
```

Comment:
`tests/` can be dropped - there's no reason we should need to ship the tests themselves in the bundle.